### DISCUSSION TITLE
Deploy Application and Claim to multiple clusters using same underlying resource

### DISCUSSION THREAD

**Discussion by florianmutter** [crossplane/crossplane#5851] [Q&A]
I would like to deploy my application to multiple clusters at the same time. I would also like to have the claim for a resource next to the deployment. That would mean it would also be deployed to multiple clusters. Is it possible to only have one underlying resource that is referenced by both claims in different clusters?

I found the `Observer` management policy that would allow me to create a database in one cluster but only observe it in the other. I'm not sure though how that would work in practice.

**Comment by vibe** [comment]
You can definitely use the `observe` policy to have the same resource represented in multiple clusters, I have done this a couple of times, mostly for troubleshooting or representing a remote resource in a local cluster.

For an automated workflow depending on the resource you need a deterministic way of computing the `external-name` to allow crossplane to import the existing state. For some resources this is simple, for example for an RDS database cluster in AWS, the external name is just the database name, other resources may require a full arn. If you can compute it easily that's great, otherwise you'll have to do some sort of dynamic lookup to fetch it. (cloud api, cluster api, etc)
